### PR TITLE
Fix stream location for StreamHandler

### PR DESCRIPTION
**Pull request by bradjones1**
This should read `stream` instead of `location`.

### Pull Request Checklist

* [x] Pull request is based on the develop branch

This is a docs-only change.

### CODE REVIEW

**Comment by babolivier** [comment]
Hi @bradjones1, and thanks for your PR!

> This is a docs-only change.

Sorry, but we also require a sign-off and changelog file for documentation changes. Could you add them please?

File: changelog.d/11946.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixes incorrect yaml key in structured logging example.
```
Comment (clokep) [inline changelog.d/11946.doc:1]:
```suggestion
Correct the structured logging configuration example. Contributed by Brad Jones.
```

**Review by clokep — APPROVED** [review]
Thank you!

**Comment by clokep** [comment]
Looks great, thank you so much for fixing it!

**Pull request merged**